BlueTeamFigures.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedBallTracker
+{
+    class BlueTeamFigures
+    {
+        //TODO Karolis Indexed properties for demo purposes, not really useful
+        private string[] figuresNames = new string[5]
```

Comment:
This is class properties for getters and setters? @droplt ?
Are these names just for sake of implementation? :D

Reply:
These names is just for implementation, to show how indexed properties work ;)